src/docs/references/03-email-templates/index.md

```diff
@@ -331,22 +331,28 @@ Context for transaction emails:
     "id": "uuid",
     "first-name": "string",
     "last-name": "string",
-    "display-name": "string"
+    "display-name": "string",
+    "private-data": "extended-data",
+    "public-data": "extended-data",
+    "protected-data": "extended-data"
   },
   "marketplace": {
     "name": "string",
-    "url" "string"
+    "url": "string"
   },
   "recipient-role": {"one-of": ["provider", "customer"]},
```

Comment:
Could you change these `one-of` things? I think the syntax is really weird and could be a lot simpler:

```suggestion
   "recipient-role": "provider", // either "provider" or "customer"
```

Reply:
@kpuputti or what about
```
"recipient-role": "string", // either "provider" or "customer"
```
?

Reply:
Changing to the above to get this forward.